source/views/home/button.tsx

```diff
@@ -16,23 +17,32 @@ export function HomeScreenButton({view, onPress}: Props): JSX.Element {
 	let foreground =
 		view.foreground === 'light' ? styles.lightForeground : styles.darkForeground
 
-	return (
-		<SafeAreaView>
-			<Touchable
-				accessibilityLabel={view.title}
-				accessibilityRole="button"
-				accessible={true}
-				highlight={false}
-				onPress={onPress}
-				style={[styles.button, {backgroundColor: view.tint}]}
-			>
-				<View style={styles.contents}>
-					<Icon name={view.icon} size={32} style={[foreground, styles.icon]} />
-					<Text style={[foreground, styles.text]}>{view.title}</Text>
-				</View>
-			</Touchable>
-		</SafeAreaView>
+	const Internaltouchable = () => (
+		<Touchable
+			accessibilityLabel={view.title}
+			accessibilityRole="button"
+			accessible={true}
+			activeOpacity={0.9}
+			highlight={false}
+			onPress={onPress}
+			style={[styles.button, {backgroundColor: view.tint}]}
+		>
+			<View style={styles.contents}>
+				<Icon name={view.icon} size={32} style={[foreground, styles.icon]} />
+				<Text style={[foreground, styles.text]}>{view.title}</Text>
+			</View>
+		</Touchable>
 	)
+
+	return Platform.select({
+		ios: <Internaltouchable />,
+		android: (
+			<SafeAreaView>
+				<Internaltouchable />
+			</SafeAreaView>
+		),
+		default: <></>,
+	})
```

Comment:
PascalCase the type name? i.e. `InternalTouchable`
```suggestion
	const InternalTouchable = () => (
		<Touchable
			accessibilityLabel={view.title}
			accessibilityRole="button"
			accessible={true}
			activeOpacity={0.9}
			highlight={false}
			onPress={onPress}
			style={[styles.button, {backgroundColor: view.tint}]}
		>
			<View style={styles.contents}>
				<Icon name={view.icon} size={32} style={[foreground, styles.icon]} />
				<Text style={[foreground, styles.text]}>{view.title}</Text>
			</View>
		</Touchable>
	)

	return Platform.select({
		ios: <Internaltouchable />,
		android: (
			<SafeAreaView>
				<InternalTouchable />
			</SafeAreaView>
		),
		default: <></>,
	})
```